drivers/gpio/gpio_nrfx.c

```diff
@@ -280,7 +367,7 @@ static int gpio_nrfx_pin_manage_callback(struct device *port,
 	}
 
 	for (u8_t curr_pin = from_pin; curr_pin <= to_pin; ++curr_pin) {
-		WRITE_BIT(data->int_en, curr_pin, enable);
+		WRITE_BIT(data->pin_int_en, curr_pin, enable);
```

Comment:
I have one doubt here. After `int_en` is removed and only `pin_int_en` is used to control the enabling of interrupts, it is possible to enable an interrupt with undefined configuration by calling only `gpio_pin_enable_callback`, and I guess it should not be allowed. Or did I miss something?

Reply:
I think we should stay with two variables for now and switch to one when this deprecated function is removed.

Reply:
I agree, moving to one variable was a bit premature. I reverted back to the old design. One difference is that now interrupts are enabled as soon as `gpio_pin_configure` or `gpio_pin_interrupt_configure` function is called - in agreement with the new API design - there is no need to call `gpio_pin_enable_callback` additionally.